package.json

```diff
@@ -20,7 +20,7 @@
         "connect-requestid": "^1.1.0",
         "cors": "^2.8.4",
         "dataloader": "^1.4.0",
-        "express": "^4.16.3",
```

Comment:
Just want to verify that we want 4.17.3 and not ^4.17.3

Reply:
Good call, it looks like we can. This gets us express 4.18.2, [which has qs 6.11.0](https://github.com/expressjs/express/blob/4.18.2/package.json#L52) which has [the fix](https://github.com/ljharb/qs/blob/v6.11.0/lib/parse.js#L139-L141) (that came from [here](https://github.com/ljharb/qs/pull/428)). And I verified that I can't reproduce the vulnerability with this new version of nodule-express + express.

(I'm somewhat obsessively double-checking all this because I spent the better part of the day jumping through dependency hell to test this).